.github/workflows/sync-malicious-packages.yaml

```diff
@@ -0,0 +1,46 @@
+name: Deploy
+
+on:
+  push:
+    branches:
+      - main
+  workflow_dispatch:
+
+  
+permissions:
+  contents: write
+  id-token: write
+
+jobs:
+  synchronize-malicious-packages:
+    name: 'Synchronize malicious packages'
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - uses: actions/setup-python@v5
+        with:
+          python-version: '3.12' 
+
+      - name: Install AWS CLI
+        run: |
+          curl https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip -o awscliv2.zip
+          unzip awscliv2.zip
+          sudo ./aws/install
+
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v4
+        with:
+          aws-region: eu-west-3
+          role-to-assume: ${{ secrets.IAM_ROLE_ARN }}
+          role-session-name: github-actions
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install -r scripts/sync-malicious-packages/requirements.txt
+
+      - name: Synchronize malicious packages
+        run: |
+          python scripts/sync-malicious-packages/sync-malicious-packages.py --since 2024-01-01 --destination ./samples/pypi --s3-bucket ${{ secrets.S3_BUCKET_NAME }} --dynamodb-table ${{ secrets.DYNAMODB_TABLE_NAME }} --ecosystem pypi
```

Comment:
```suggestion
          python scripts/sync-malicious-packages/sync-malicious-packages.py --since $(date -d "2 month ago" +%d-%m-%Y) --destination ./samples/pypi --s3-bucket ${{ secrets.S3_BUCKET_NAME }} --dynamodb-table ${{ secrets.DYNAMODB_TABLE_NAME }} --ecosystem pypi
```

Reply:
TIL, nice one - I was thinking we'd just sync anything from 2024 in case we retroactively mark some old packages as malicious

Reply:
Yeah, I thought about it, IMO we should keep a smaller window.
This will be scheduled and I think sync costs are unnecessary.
We can schedule another job that runs 1 time every month or so to achieve that with low impact

Reply:
sounds good, keep in mind that this is only used in the DynamoDB request so whatever we use shouldn't be too expensive. I'll use your 2 months window for now and we can revisit

Reply:
done in [1c71dc9](https://github.com/DataDog/malicious-software-packages-dataset/pull/10/commits/1c71dc9b76783f31a6c836eea96ec12b41000ead)